app-web/plugins/gatsby-source-github-all/__tests__/registryHelpers.js

```diff
@@ -37,7 +37,7 @@ const SOURCE_3 = {
   },
 };
 
-const registry = [
+export const registry = [
```

Comment:
let's not move __fixtures__ outside of that directory. This shoould be the other way around if you want to reduce duplication.